plugins/modules/helm.py

```diff
@@ -432,8 +432,8 @@ def main():
             # Helm options
             disable_hook=dict(type='bool', default=False),
             force=dict(type='bool', default=False),
-            kube_context=dict(type='str', aliases=['context'], fallback=(env_fallback, ['K8S_AUTH_CONTEXT'])),
-            kubeconfig_path=dict(type='path', aliases=['kubeconfig'], fallback=(env_fallback, ['K8S_AUTH_KUBECONFIG'])),
+            context=dict(type='str', aliases=['kube_context'], fallback=(env_fallback, ['K8S_AUTH_CONTEXT'])),
+            kubeconfig=dict(type='path', aliases=['kubeconfig_path'], fallback=(env_fallback, ['K8S_AUTH_KUBECONFIG'])),
```

Comment:
We already have these as aliases. I am trying to understand the effect of this change.

Reply:
In https://github.com/ansible-collections/community.kubernetes/pull/319, I need to access the options in the different modules. I would like them to have the same name in all the modules.
The current lack of consistency complecify the code base and was the source of a bug in my PR.

Reply:
OK. Got it.